frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -434,7 +431,31 @@ export const GovInfoFormContainer: React.FunctionComponent = () => {
       return;
     }
     await handleUpdateUser(meId, data)
-      .then(() => {
+      .then((result) => {
+        if (result.isProfileComplete) {
+          if (result.isProfileComplete) {
+            sessionStorage.setItem("currentProfileStatus", "Complete");
+          } else {
+            sessionStorage.setItem("currentProfileStatus", "InComplete");
+          }
+          const preProfileStatus = sessionStorage.getItem("preProfileStatus");
+          const currentProfileStatus = sessionStorage.getItem(
+            "currentProfileStatus",
+          );
+
+          if (preProfileStatus === "InComplete") {
+            if (currentProfileStatus === "Complete") {
+              toast.success(
+                intl.formatMessage({
+                  defaultMessage:
+                    "All required fields are complete. You can now change your status.",
+                  description:
+                    "Message displayed to user when user profile completed.",
+                }),
+              );
+            }
+          }
+        }
```

Comment:
Could you add some jest tests, too?  Maybe:
- If the page is changed and the profile goes from incomplete to complete a toast pops
- If the page is changed the profile stays incomplete then no toast
- If the page is changed and the profile stays complete then no toast

Reply:
Since we are displaying the toast in all the necessary forms, Test needs to be done in all the forms. Am I correct?